### DISCUSSION TITLE
Create AETitle

### DISCUSSION THREAD

**Discussion by esefberfu** [dcm4che/dcm4chee-arc-light#3167] [Q&A] [answered]
Hello,
There are two aetitle that names are NAPPACS and TESTAE on my dcm4chee server.
I can send dicom successfully to both aetitle.

Example:
NAPPACS <- STUDY A
TESTAE <- STUDY B
i have sentSTUDY A to NAPPACS and STUDY B to TESTAE.

if I search(CFIND) any study on any aetitle like TESTAE or NAPPACS, DCM4CHEE returns all study on my pacs server independently from aetitle

**Comment by gunterze** [comment, marked as answer]
If you want to restrict access dependent on the Archive AE see [Access control based on Archive AEs](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Access-control-based-on-Archive-AEs)
